go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -975,6 +975,9 @@ var defaultConfig = TabletConfig{
 		Mode:              Disable,
 		HeartbeatInterval: 250 * time.Millisecond,
 	},
+	GracePeriods: GracePeriodsConfig{
+		Shutdown: 3 * time.Second,
+	},
 	HotRowProtection: HotRowProtectionConfig{
 		Mode: Disable,
```

Comment:
This default should not be `3 seconds` as the transaction and query timeout itself is `30 seconds`. This grace period should be `30 seconds` + `delta` to account for other default values.

Reply:
@deepthi What do you think?

Reply:
We are changing from `infinite` to `3 seconds` is a drastic value change for me. Can impact user's databases with unknowns.

Reply:
I think 3 seconds is fine. We will document this as a breaking change. This is just how much you are allowing for already open queries and transactions to complete. In most cases they won't be running for the full 30 seconds or whatever that value is set to.
It is more important for PRS to complete in a timely manner because new queries get blocked as long as PRS is running. It's a tradeoff that people can make on their own.

Reply:
Sounds good